src/components/menus/LocalDocsMenu.ts

```
@@ -0,0 +1,683 @@
+        // Clear current list
+        this.documentListEl.innerHTML = '';
+        
+        try {
+            // Get all documents
+            const documents = this.manager?.getAllDocuments() || [];
+            
+            // Filter if search term is provided
+            const filteredDocs = searchTerm 
+                ? documents.filter(doc => 
+                    doc.title.toLowerCase().includes(searchTerm) ||
+                    (doc.tags && doc.tags.some(tag => tag.toLowerCase().includes(searchTerm))) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)